### PR TITLE
config changes

### PR DESCRIPTION
**Pull request by felipemello1**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [x] fix a bug
- [ ] update tests and/or documentation
- [ ] other (please add here)

#### Changelog
3 mains fixes:
1) llama 3.1 KD config loading wrong checkpoint
2) qwen2 config loading wrong checkpoint
3) llama 70b repeated fused argument + warning that it doesnt work on torch stable
4) llama 405B-lora warning to not use pytorch stable

extra:
added compile flag for llama. I should have done it in another PR, sorry/not sorry.

#### Test plan
ran the configs and saved ckpt

### CODE REVIEW

File: recipes/configs/llama3/70B_full.yaml [hunk 1]
```diff
@@ -14,6 +14,12 @@
 #   tune run --nproc_per_node 8 full_finetune_distributed --config llama3/70B_full checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
 #
 # This config is only tested on an 8xA100 machine.
+#
+# !!!!!!!!!!!!!
+# !!!!!!!!!!!!!
```
Comment (RdoubleA) [inline recipes/configs/llama3/70B_full.yaml:19]: nit: I would take the !!!! out personally

File: recipes/configs/qwen2/knowledge_distillation_single_device.yaml [hunk 2]
```diff
@@ -44,9 +44,9 @@ checkpointer:
...
 teacher_checkpointer:
   _component_: torchtune.training.FullModelHFCheckpointer
-  checkpoint_dir: /tmp/Qwen2-1.5B-Instruct-lora-finetune
+  checkpoint_dir: /tmp/Qwen2-1.5B-Instruct
   checkpoint_files: [
-    hf_model_0001_0.pt
+    model.safetensors
   ]
   recipe_checkpoint: null
   output_dir: /tmp/Qwen2-1.5B-Instruct-lora-finetune
```
Comment (RdoubleA) [inline recipes/configs/qwen2/knowledge_distillation_single_device.yaml:52]: should change output dir here

**Review by RdoubleA — APPROVED** [review]

File: recipes/configs/llama3/70B_full.yaml (hunk 1 above)
Comment (felipemello1) [reply recipes/configs/llama3/70B_full.yaml:19]: i am afraid that users would ignore it without the "!", and not understand why it crashes

File: recipes/configs/qwen2/knowledge_distillation_single_device.yaml (hunk 2 above)
Comment (felipemello1) [reply recipes/configs/qwen2/knowledge_distillation_single_device.yaml:52]: i thought about it. Indeed in all of our configs we use the same output dir. I will make the change

**Pull request merged**